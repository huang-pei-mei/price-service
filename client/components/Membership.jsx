import React from 'react';

const Membership = () => (
  <div id='membership'>
    <div id='premiumPlus' className='priceBody'>
      Included with <span id='premium'>premium</span><span id='plus'>plus</span>
    </div>
    <button className='priceButton priceBody' id='memberEnroll'>Try for $0.00</button>
    <div id='memberBenefits' className='priceBody'>
      <ul>
        <li>1 audiobook of your choice.</li>
        <li>Stream or download thousands of included titles.</li>
        <li>$14.95 a month after 30 day trial. Cancel anytime.</li>
      </ul>
    </div>
  </div>
);

export default Membership;