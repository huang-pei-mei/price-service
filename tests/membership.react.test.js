import React from 'react';
import { shallow } from 'enzyme';
import Membership from '../client/components/Membership.jsx';

describe('Membership', () => {
  it('should render my component', () => {
    const component = shallow(<Membership />);

    expect(component.getElements()).toMatchSnapshot();
  });
});