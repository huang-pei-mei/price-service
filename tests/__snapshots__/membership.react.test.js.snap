// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Membership should render my component 1`] = `
Array [
  <div
    id="membership"
  >
    <div
      className="priceBody"
      id="premiumPlus"
    >
      Included with 
      <span
        id="premium"
      >
        premium
      </span>
      <span
        id="plus"
      >
        plus
      </span>
    </div>
    <div
      className="priceBody"
      id="memberBenefits"
    >
      You'll have access to these benefits
      <ul>
        <li>
          1 audiobook of your choice.
        </li>
        <li>
          Stream or download thousands of included titles.
        </li>
        <li>
          $14.95 a month after 30 day trial. Cancel anytime.
        </li>
      </ul>
    </div>
    <button
      className="priceButton"
      id="memberEnroll"
    >
      Continue for free
    </button>
  </div>,
]
`;
